src/pages/Login/Login.css

```diff
@@ -56,7 +56,7 @@ text-align: center;
 
 /* .down-image{
 /* height: auto; */
- */
+ 
```

Comment:
Check this one also @JalandharBandari

Reply:
it was a type error please remove @TejaBonam